chrome/content/zotero/xpcom/data/collections.js

```diff
@@ -46,13 +46,23 @@ Zotero.Collections = function() {
 		hasChildCollections: "(SELECT COUNT(*) FROM collections WHERE "
 			+ "parentCollectionID=O.collectionID) != 0 AS hasChildCollections",
 		hasChildItems: "(SELECT COUNT(*) FROM collectionItems WHERE "
-			+ "collectionID=O.collectionID) != 0 AS hasChildItems"
+			+ "collectionID=O.collectionID) != 0 AS hasChildItems",
```

Comment:
hmm, I looks like this may have gotten left over after a merge conflict. Will clean it up.